vpr/src/place/move_transactions.cpp

```diff
@@ -1,7 +1,17 @@
 #include "move_utils.h"
 
 #include "globals.h"
-#include "place_util.h"
+
+t_pl_blocks_to_be_moved::t_pl_blocks_to_be_moved(size_t max_blocks){
+        moved_blocks.reserve(max_blocks);
+        moved_blocks.resize(0);
+}
+
+size_t t_pl_blocks_to_be_moved::get_size_and_increment() {
+    VTR_ASSERT(moved_blocks.size() < moved_blocks.capacity());
```

Comment:
I'd make this a VTR_ASSERT_SAFE as it is in a hot routine (VTR_ASSERT_SAFE is taken out of release build).